src/uu/ls/src/ls.rs

```diff
@@ -603,10 +648,49 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
                 .short("F")
                 .long(options::CLASSIFY)
                 .help("Append a character to each file name indicating the file type. Also, for \
-                    regular files that are executable, append '*'. The file type indicators are \
-                    '/' for directories, '@' for symbolic links, '|' for FIFOs, '=' for sockets, \
-                    '>' for doors, and nothing for regular files.",
-        ))
+                       regular files that are executable, append '*'. The file type indicators are \
+                       '/' for directories, '@' for symbolic links, '|' for FIFOs, '=' for sockets, \
+                       '>' for doors, and nothing for regular files.")
+                .overrides_with_all(&[
+                    options::FILE_TYPE,
+                    options::SLASH,
+                    options::CLASSIFY,
+                    options::INDICATOR_STYLE,
+                    options::indicator_style::NONE,
+                    options::indicator_style::SLASH,
+                    options::indicator_style::FILE_TYPE,
+                    options::indicator_style::CLASSIFY,
```

Comment:
It's not necessary to let have these override with the `options::indicator_style::*`, since those are not names of the `Arg`s. Same goes for the other options you added.